common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -16,6 +16,79 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     document
   }
 
+  private def documentWithVideos(videoUrls: Seq[String]): Document = {
```

Comment:
Nitpicking: you can use `varargs` so you don't have to pass a `Seq` but multiple `Strings`
ie:
```
private def documentWithVideos(videoUrls: String*): Document
....
documentWithVideos("https://vimeo.com/")
documentWithVideos("https://www.youtube.com/watch?v=foo_12-34","https://vimeo.com/1234")

```